src/index.html

```diff
@@ -3,9 +3,18 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+    <title>ToDoList</title>
+    <link rel="stylesheet" type="text/css" href="./style.css">
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+  <div class="container">
+    <div class="demo">Demo</div>
+    <input class="add-task" placeholder="Add to your list...">
+    <ul class="to-do-list">
+
+    </ul>
+    <button class="clear-completed">Clear all completed</button>
+  </div>
+  <script type="module" src="/src/modules/index.js"></script>
 </body>
```

Comment:
- [ ] As the previous reviewer suggested kindly make sure to fix your design so it looks exactly like the given design, as for now your design looks like this 👇🏼 

![Screenshot_20230621_015731](https://github.com/EvansSnave/to-do-list/assets/25227022/0038f7b1-88a1-4492-bb35-b3f28ceed291)